activerecord/lib/active_record/locale/en.yml

```diff
@@ -17,7 +17,7 @@ en:
         record_invalid: "Validation failed: %{errors}"
         restrict_dependent_destroy:
           one: "Cannot delete record because a dependent %{record} exists"
-          many: "Cannot delete record because dependent %{record} exist"
+          other: "Cannot delete record because dependent %{record} exist"
```

Comment:
Thanks, unfortunately this is not about I18n's pluralization but about has one/many, as you can see [in the handling logic from Active Record](https://github.com/rails/rails/blob/ea97c763c18b4f7534589665c73d501fcec229a3/activerecord/lib/active_record/associations/has_many_association.rb#L19). The [build failure](https://travis-ci.org/rails/rails/jobs/58292541#L366-L371) shows exactly that.